### ISSUE TITLE
Improve the upload form user experience

### ISSUE THREAD

**Issue by nleroy917** [pepkit/pephub#70]
The upload form as is, works. However, it could be better. To create a better user experience, 3 things should be changed:
1. Allow the user to drag files onto the browser
2. Validate the PEP as it comes in before submission
3. Allow the user to add a PEP to an organization

**Comment by nleroy917** [comment]
I'm working on this. @nsheff you said you have some code for this? It doesn't seem hard, but if you already have something.

Further, whats the serverside code look like for parsing an arbitrary list of files?

**Comment by nsheff** [comment]
For the drag-and-drop, I'll look this up tonight and send you (I have something on another computer somewhere).

For the arbitrary list of files, here's an example I wrote for eido: https://github.com/pepkit/eido/blob/536d944206d031b93e1cceac5585f12d9114614a/webeido/webeido.py#L107-L112

**Comment by nleroy917** [comment]
I see this:
 ```python
if ext == ".yaml" or ext == ".yml":
       pconf = uploaded.name
```

Doesn't that mean a config file is required on upload? I thought you could instantiate peppy projects with just a sample table csv?

**Comment by nleroy917** [comment]
I had this flow in my mind. I might be over thinking this...

```mermaid
graph TD;
 A[User uploads files] --> B{Included `.yaml` file?}
 B -- Yes --> C[Init project using `file.yaml`]
B -- No --> D{Included `.csv` file?}
D -- No --> E[Return `400`]
D -- Yes --> F[Init project from `csv`]
```

**Comment by nsheff** [comment]
> I see this:
> 
> ```python
> if ext == ".yaml" or ext == ".yml":
>       pconf = uploaded.name
> ```
> 
> Doesn't that mean a config file is required on upload? I thought you could instantiate peppy projects with just a sample table csv?

This was just a prototype. It may even have been written before we finalized the csv-only option. I think your expected flowchart is correct.

**Comment by nleroy917** [comment]
This should be solved with `v0.5.0` release #80
